cmd/github-labels/README.md

```diff
@@ -0,0 +1,75 @@
+* [Overview](#overview)
+* [Generating the combined labels database](#generating-the-combined-labels-database)
+* [Checking and summarising the labels database](#checking-and-summarising-the-labels-database)
+    * [Show labels](#show-labels)
+    * [Show categories](#show-categories)
+    * [Check only](#check-only)
+    * [Full details](#full-details)
+
+# Overview
+
+The Kata Project uses a number of GitHub repositories. To allow issues and PRs
+to be handled consistently between repositories a standard set of issue labels
+are used. These labels are stored in YAML format in the master [labels
+database template](labels.yaml.in). This file is human-readable and
+machine-readable and self-describing (take a look at the file for the
+introductory description).
+
+Additionally, each repository can contain a set of additional
+(repository-specific) labels which are stored in a top-level YAML template
+file called `labels.yaml.in`.
+
+Expanding the templates and merging the two databases describes the full set
+of labels a repository uses.
+
+# Generating the combined labels database
+
+The combined labels database is generated by running the
+`github_labels.sh` script with the `generate` parameter, also passing the
+repository to generate the combined labels database for and the name of a file
+to write the combined database to:
+
+```sh
+$ ./github-labels.sh generate github.com/kata-containers/runtime /tmp/combined.yaml
+```
+
+The script validates the combined labels database by performing a number of
```

Comment:
Lines 36-39 suggested rewrite: 
This script validates the combined labels database by performing a number of checks, including running the `kata-github-labels` tool in checking mode. See the [Checking and summarising the labels database](#checking-and-summarising-the-labels-database) section for more information.